app/code/community/Nexcessnet/Turpentine/Model/Observer/Esi.php

```diff
@@ -325,8 +325,6 @@ protected function _getEsiData( $blockObject, $esiOptions ) {
         $methodParam = $esiHelper->getEsiMethodParam();
         $esiData = new Varien_Object();
         $esiData->setStoreId( Mage::app()->getStore()->getId() );
-        $esiData->setDesignPackage( Mage::getDesign()->getPackageName() );
```

Comment:
This is being used at https://github.com/nexcess/magento-turpentine/blob/c9a003a95a49628dec502646f1f3021c304f783e/app/code/community/Nexcessnet/Turpentine/controllers/EsiController.php#L187